src/common/types/navigation/params/root-stack-param.ts

```diff
@@ -7,6 +7,7 @@ type RootStackParamList = {
   [AppRoute.APP]: NavigatorScreenParams<AppStackParamList>;
   [AuthRoute.SIGN_IN]: undefined;
   [AuthRoute.SIGN_UP]: undefined;
+  [AuthRoute.COMPLETE_REGISTRATION]: undefined;
```

Comment:
This screen has token as param
```suggestion
  [AuthRoute.COMPLETE_REGISTRATION]: {
       token: string
  };
```